docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
None, please. Any other value calls attention to itself and invites the question "now why isn't that None?".